package.json

```diff
@@ -6,6 +6,8 @@
   "description": "An automation task to compile css-modules and remove it from the components",
   "scripts": {
     "test": "NODE_ENV=test mocha --compilers js:babel-register --reporter emoji-reporter ./test/**/*.spec.js",
+    "coverage": "NODE_ENV=test nyc --cache --require babel-core/register mocha ./test/**/*.spec.js --timeout=8000",
```

Comment:
I tried to run this command as `NODE_ENV=test nyc npm test` and I didn't get any errors, can you please give me more info about the emoji issue?